lib/razor/data/repo.rb

```diff
@@ -170,7 +190,8 @@ def filesystem_safe_name
     # done, notify ourselves of that so any cleanup required can be performed.
     def unpack_repo(command, path)
       destination = iso_location
-      destination.mkpath        # in case it didn't already exist
+      FileUtils.remove_entry_secure(destination) if destination.directory? #delete the destination and contents if it already exists
```

Comment:
This has failed in the past if any files in the repo are not extracted with write permissions. The fix, already implemented, is to recursively set this privilege before attempting the deletion.